Packs/Infoblox/Integrations/Infoblox/Infoblox.py

```diff
@@ -330,15 +471,248 @@ def parse_demisto_exception(error: DemistoException, field_in_error: str = 'text
+    [{"Site": "Tel-Aviv"}]
+
+    >>>> transform_ext_attrs("IB Discovery Owned=EMEA,Site=Tel-Aviv")
+    [{"*IB Discovery Owned": "EMEA", "*Site": "Tel-Aviv"}]
+    ```
+    """
+
+    # In case there are no delimiters present in the input
+    if "," not in ext_attrs and "=" not in ext_attrs:
+        return []
+
+    l_ext_attrs: list[dict] = []
+
+    attributes = ext_attrs.split(",")
+
+    for ext_attr in attributes:
+        try:
+            key, value = ext_attr.split("=")
+            if key and value:
+                l_ext_attrs.append({f"*{key.strip()}": value.strip()})
+        except ValueError:
+            demisto.error(f"Unable to parse ext_attrs: {ext_attrs}")
+            continue
+
+    return l_ext_attrs
+
+
+def valid_ip(ip: str):
+    """Validate IP address format"""
+
+    try:
+        ipaddress.IPv4Address(ip)
+    except ValueError:
+        raise InvalidIPAddress(f"'{ip}' is not a valid IPv4 address")
+
+
+def valid_netmask(address: str):
+    """Validate netmask format"""
+
+    try:
+        ipaddress.ip_network(address, strict=False)
+    except ValueError:
+        raise InvalidNetmask(f"'{address}' is not a valid netmask")
+
+
+def valid_ip_range(from_ip: str, to_ip: str):
+    """Validate IP range format"""
+    try:
+        from_address = ipaddress.IPv4Address(from_ip)
+        to_address = ipaddress.IPv4Address(to_ip)
+        list(ipaddress.summarize_address_range(from_address, to_address))
+    except ValueError as err:
+        raise InvalidIPRange(f"'{from_ip}' to '{to_ip}' is not a valid IPv4 range: {err}")
+
+
+def transform_ipv4_range(from_ip: str, to_ip: str) -> dict[str, str]:
+    """Transform IPv4 range to list of IPs.
+
+    Args:
+        from_ip: Start of IPv4 range.
+        to_ip: End of IPv4 range.
+
+    Returns:
+        dictionary of IPv4 addresses in range.
+    """
+
+    return {"ip_address>": from_ip, "ip_address<": to_ip}
+
+
+def transform_return_fields(additional_fields: str) -> dict[str, Any]:
+    """
+    Helper function to transform the return fields into a
+    dictionary that we can feed into the client as request
+    parameters.
+
+    Arguments:
+    - `additional_fields` (``str``): A comma-separated list of additional fields.
+
+    Returns:
+    - `dict[str, Any]` request parameters
+    """
+
+    result = {}
+
+    if additional_fields:
+        result[InfoBloxNIOSClient.REQUEST_PARAM_RETURN_FIELDS_KEY] = additional_fields
+
+    return result
+
+
+def transform_network_info_context(network_info: list[dict[str, Any]]) -> list[dict[str, Any]]:
+    """
+    Helper function to transform the network info
+    raw response to the expected context structure.
+
+    Args:
+    - `network_info` (``list[dict[str, Any]]``): The network info request result.
+
+    Returns:
+    - `list[dict[str, Any]]` context output.
+    """
+
+    output: list[dict[str, Any]] = []
+    additional_options: list[dict[str, Any]] = []
+
+    for network in network_info:
```

Comment:
Why remove the previous implementation of dictionary translation? It is easier and cleaner:
```python
key_name = my_dict[k]
i[key_name] = v
```

Reply:
https://github.com/demisto/content/pull/32619/commits/430564459d8dfeb96f0465bf08a39e74db67385f